pkg/artifact/jsonnetbundler/package.go

```diff
@@ -0,0 +1,126 @@
+package jsonnetbundler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+
+	"github.com/neticdk/go-common/pkg/artifact"
+	"github.com/pkg/errors"
+
+	"github.com/jsonnet-bundler/jsonnet-bundler/pkg"
+	"github.com/jsonnet-bundler/jsonnet-bundler/pkg/jsonnetfile"
+	specv1 "github.com/jsonnet-bundler/jsonnet-bundler/spec/v1"
+	"github.com/jsonnet-bundler/jsonnet-bundler/spec/v1/deps"
+)
+
+const filePermissionNewFile = 0o640
+
+type PackageOptions struct {
+	JsonnetHome string
+}
+
+// Install installs a package and its dependencies in the vendor directory.
+// It mimics https://github.com/jsonnet-bundler/jsonnet-bundler/blob/master/cmd/jb/install.go
+func Install(_ context.Context, a *artifact.Artifact, opts *PackageOptions) (*artifact.PullResult, error) {
+	if opts == nil {
+		opts = &PackageOptions{}
+	}
+
+	if opts.JsonnetHome == "" {
+		opts.JsonnetHome = "vendor"
+	}
+
+	jbfilebytes, err := os.ReadFile(filepath.Join(a.BaseDir, jsonnetfile.File))
+	if err != nil {
+		return nil, fmt.Errorf("reading %q: %w", jsonnetfile.File, err)
+	}
+
+	jsonnetFile, err := jsonnetfile.Unmarshal(jbfilebytes)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshalling %q: %w", jsonnetfile.File, err)
+	}
```

Comment:
This seems to be the same as `jsonnetfile.Load`?
```suggestion
	jsonnetFile, err := jsonnetfile.Load(filepath.Join(a.BaseDir, jsonnetfile.File))
	if err != nil {
		return nil, fmt.Errorf("loading %q: %w", jsonnetfile.File, err)
	}
```

Reply:
`jsonnetfile.Load` results in a `v1.Jsonfile` and we are using `jbfilebytes` to re-write the jsonnetfile. I think that's why they don't use it in the install command.